### PR TITLE
Add elasticdl_preprocessing layers in package data

### PR DESCRIPTION
**Pull request by workingloong**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -28,6 +28,7 @@
             "Makefile",
             "requirements.txt",
             "pkg/kernel/capi/*",
+            "elasticdl_preprocessing/layers/*",
```
Comment (brightcoder01) [inline setup.py]: Why not elasticdl_preprocessing/*

Comment (workingloong) [reply setup.py]: "package_data" can only contain files. Here, we can use "package_dir", https://docs.python.org/2/distutils/setupscript.html#listing-whole-packages

**Review by brightcoder01 — APPROVED** [review]

**Review by brightcoder01 — COMMENTED** [review]
Will accept if the CI pass.

**Comment by workingloong** [comment]
There is `__init__.py` in the elasticdl_preprocessing directory and the python files in elasticdl_preprocessing will be packaged when building a wheel. So, we don't need the configuration.

**Pull request closed without merging**